### PR TITLE
Update docs for configuring Apache airflow

### PR DESCRIPTION
**Pull request by sanika93**
Proposed changes -

1. For installing apache airflow on kubernetes the following parameters needed to be updated in the sample helm airflow values.yaml file - 
    i. flower.enabled: false
    ii. redis.enabled: false

2. Added notes which will help first time users install airflow successfully. 

### How was this pull request tested?

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by sanika93** [comment]
@akchinSTC FYI

File: docs/source/recipes/configure-airflow-as-a-runtime.md [hunk 1]
```diff
@@ -80,8 +80,8 @@ To deploy Apache Airflow on a new Kubernetes cluster:
    ```
   
 2. Download, review, and customize the [sample `helm` configuration](https://raw.githubusercontent.com/elyra-ai/elyra/master/etc/kubernetes/airflow/helm/values.yaml) (or customize an existing configuration):
```
Comment (akchinSTC) [inline docs/source/recipes/configure-airflow-as-a-runtime.md:82]: Lets add a blurb here letting folks know that this sample configuration will use the `KubernetesExecutor` by default.

**Review by akchinSTC — COMMENTED** [review]
Thanks for improving our docs!

**Comment by karlaspuldaro** [comment]
@akchinSTC does this also fix #1978 ?

File: docs/source/recipes/configure-airflow-as-a-runtime.md (hunk 1 above)
Comment (sanika93) [reply docs/source/recipes/configure-airflow-as-a-runtime.md]: done 👍

**Review by akchinSTC — APPROVED** [review]

**Pull request merged**